### ISSUE TITLE
Is it possible to set keras layer output ?

### ISSUE THREAD

**Issue by dodgerV** [fchollet/keras#8261]
Hello!
I need to modify output of last feature map of my second convolution layer. 
Or add array to my conv layer output if it's possible.
Below is  python script i created and example of desired change in output.
Thank you for your help!

import numpy as np
from keras import backend as K

num=18
m=11 
n=50
k=3
np.random.seed(100)
features = np.random.rand(num,m,n,k)

#model
input_shape=features.shape[1:]
model = Sequential()
model.add(Conv2D(2, kernel_size=(1, 3), strides=(1, 1),activation='relu',input_shape=input_shape,padding="valid"))
model.add(Conv2D(21, kernel_size=(1, 48), strides=(1, 1),padding="valid",activation='relu'))
model.add(Conv2D(1, kernel_size=(1, 1), strides=(1, 1),activation='relu',padding="valid"))
model.add(Dense(1, activation='softmax'))
Adam = optimizers.Adam(lr=0.00003, beta_1=0.9, beta_2=0.999, epsilon=1e-08, decay=0.0)
model.compile(loss='mse',optimizer=Adam)

get_1st_layer_output = K.function([model.layers[0].input],
                                  [model.layers[1].output])
layer_output = get_1st_layer_output([features])

# Setting DESIRED layer_output values
# I need to do it every propagation step.
for i in range(0,11):
    layer_output[0][0][i][0][20]=0.1
    print(layer_output[0][0][i][0][20])
